pygeo/DVGeometry.py

```diff
@@ -690,6 +693,98 @@ def addGeoDVLocal(self, dvName, lower=None, upper=None, scale=1.0,
 
         return self.DV_listLocal[dvName].nVal
 
+    def addGeoDVSpanwiseLocal(self, dvName, spanIndex, axis='y', lower=None, upper=None,
+                             scale=1.0, pointSelect=None, volList=None, config=None):
+
+        if type(config) == str:
+                    config = [config]
+
+        if pointSelect is not None:
+            if pointSelect.type != 'ijkBounds':
+                pts, ind = pointSelect.getPoints(self.FFD.coef)
+            else:
+                pts, ind = pointSelect.getPoints_ijk(self)
```

Comment:
I copied that section from addGeoDVLocal, Should I add that bit of code to both?

Reply:
Mmmm.. it looks like [`pointSelect`](https://github.com/mdolab/pygeo/blob/50d109ab84a3b1f59cdb53586cbf9d8af895365e/pygeo/geo_utils.py#L1269-L1447) is used to pick a subset of the FFD control points to be used by this variable, and then the `volList` is updated (using the missing line) to include the volumes to which these points belong to.

I am really not sure which of our cases / runs actually end up using this bit of code (I think we usually provide an explicit list of point indexes rather than an object like this) but to me that line looks like it belongs both here and in `addGeoDVLocal`. 
Not sure if this is a small bug or something irrelevant though, and ofc we don't have any tests to check this.